compute/admin_guide/compliance/cloud_discovery_saas.adoc

```diff
@@ -63,27 +75,39 @@ Then configure Prisma Cloud to protect them with a single click.
 *Prerequisites:* You onboarded cloud accounts in Prisma Cloud as described https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/connect-your-cloud-platform-to-prisma-cloud/cloud-account-onboarding[here].
 
 [.procedure]
-. Open Console.
+. Log in to Prisma Cloud.
 
-. Go to *Defend > Compliance > Cloud Platforms*.
+. Select  *Compute > Manage > Cloud Accounts*.
 
-. Select the accounts to scan with the *Discovery* checkbox.
-If there are no accounts in the table, you can import Prisma Cloud onboarded accounts, by clicking on "Add" and selecting "Prisma Cloud" as provider.
+. Select the accounts to scan.
+If there are no accounts in the table, you can import Prisma Cloud onboarded accounts, using the "Add account" workflow and selecting "Prisma Cloud" as the provider.
 
-. Click *Save*.
+. Select *Bulk actions* > *Discovery configuration*
 
-. Review the scan report.
+. Enable *Cloud discovery*.
++
+image::cloud_discovery_saas.png[]
+
+. *Save* your changes.
 
-..  Go to *Monitor > Compliance > Cloud Discovery* to see the scan report in tabular format.
+. Review the scan results.
 
-..  Go to *Radar* and select *Cloud* to see the scan report in a visual format.
+..  Select *Compute > Manage > Cloud Accounts* to view the scan report in tabular format.
+... Select the Show account details icon to see the discovery scan results for resources within the cloud account.
++
+image::cloud_discovery_details_saas.png[]
+
+..  Select  *Radar > Cloud* to view the scan report in a visual format.
++
+image::cloud_discovery_radar_saas.png[]
+In the Radar you can see the details for the resources that are protected using Defenders and agentless scanning across the services in each region.
 
-..  Click *Protect* for the entities you want Prisma Cloud to scan for vulnerabilities.
+..  Select *Defend* for the entities you want Prisma Cloud to scan for vulnerabilities.
 +
-When you click *Protect*, a new scan rule is proposed.
+A new auto-defend rule is proposed.
 Select the appropriate credential, tweak the scan rule as desired, then click *Add*.
 
-..  Scan reports can viewed under *Monitor > Vulnerabilities > {Registry|Functions}*.
+..  See the scan results on *Compute > Monitor > Vulnerabilities > {Images > Registry|Functions}*.
 
 
 [.task]
```

Comment:
@Aqsaxvv092, is this compliance scans still relevant? Why is it specific to AWS?

Can you PTAL and let me know what is relevant/accurate here?